cmd/telemeter-benchmark/main.go

```diff
@@ -153,11 +173,11 @@ func runCmd() error {
 				select {
 				case <-hup:
 					if err := b.Reconfigure(cfg); err != nil {
-						log.Printf("error: failed to reload config: %v", err)
+						level.Error(opt.Logger).Log("msg", fmt.Sprintf("failed to reload config: %v", err))
```

Comment:
The most canonical would be to have a KV pair with `"msg", ..., "err", ...`
```suggestion
						level.Error(opt.Logger).Log("msg", "failed to reload config", "err", err))
```